pkg/trace/api/otlp.go

```diff
@@ -517,6 +517,12 @@ func (o *OTLPReceiver) convertSpan(rattr map[string]string, lib pcommon.Instrume
 	if in.Events().Len() > 0 {
 		transform.SetMetaOTLP(span, "events", transform.MarshalEvents(in.Events()))
 	}
+	for i := range in.Events().Len() {
+		if in.Events().At(i).Name() == "exception" {
```

Comment:
should we do contains?

Reply:
from what I can see in the [otel implementation](https://github.com/open-telemetry/opentelemetry-go/blob/a2347542010c929ab30ff63529b4c7531af35334/sdk/trace/span.go#L500), the name of an exception span event should always be "exception", is there any reason not to use the built-in string comparison operator `==`?